# Config file for loading tracking parameters for the simulated ARCC League

debug: True
temporal_filtering: 0
dr_contour_area_cutoff: 120
camera_dist_from_ground: 1
camera_vertical_fov: 80
camera_horizontal_fov: 120
dr_height: 0.0254
blur_param: 15 # odd number, tune to reduce noise
#red_mask: "170, 120, 90, 180, 255, 255, 0, 50, 120, 5, 255, 255"
camera_topic: "/static_camera/image_raw"
